app/view_models/verify/base.rb

```diff
@@ -0,0 +1,88 @@
+module Verify
+  class Base
+    include Rails.application.routes.url_helpers
+
+    def initialize(error: nil, remaining_attempts:)
+      @error = error
+      @remaining_attempts = remaining_attempts
+    end
+
+    attr_reader :error, :remaining_attempts
+
+    def title
+      I18n.t("idv.titles.#{step_name}")
```

Comment:
I don't think this i18n key will work for every page. I believe the initial page of the verification process is referred to as `sessions.basic` and the step name is `sessions`.  I think there may be a disconnect between `finance` and `financials` as well.